src/task.c

```diff
@@ -10,23 +10,29 @@
 
 int              hook_init_done = 0;
 static int       num_hooks      = 0;
-static task_hook hooks[MAX_HOOKS];
+// RESOLVE: At 8 bytes per task_hook{}, we are burning up 16K of memory. Only 2 hooks
```

Comment:
@ajhconway - We should discuss and reconsider this. 

When we, eventually, move to a shared memory implementation, I suspect (expect) that this array will need to be allocated from shared memory segment. That will cost us 16K, whereas something like 64 bytes should suffice if we reduce MAX_HOOKS to 8.